crates/uv-publish/src/lib.rs

```diff
@@ -1025,7 +1025,7 @@ mod tests {
                             referer: true,
                             default_headers: {
                                 "accept": "*/*",
-                                "user-agent": "uv/0.6.15",
+                                "user-agent": "uv/0.6.16",
```

Comment:
We need to mask those, or ideally capture only `RequestBuilder`

Reply:
I've updated to capture just the `RequestBuilder`